diff --git a/node_modules/astrocite-bibtex/lib/grammar.d.ts b/node_modules/astrocite-bibtex/lib/grammar.d.ts
index babcc75..a871661 100644
--- a/node_modules/astrocite-bibtex/lib/grammar.d.ts
+++ b/node_modules/astrocite-bibtex/lib/grammar.d.ts
@@ -165,8 +165,4 @@ export interface ParseOptions {
     verbatimCommands?: string[];
 }
 
-export const AST: {
-    parse(input: string, options?: ParseOptions): AST;
-};
-
-export function parse(input: string): CSL.Data[];
+export function parse(input: string, options?: ParseOptions): AST;
