--- node_modules/astrocite-bibtex/index.d.ts	1985-10-26 09:15:00.000000000 +0100
+++ node_modules/astrocite-bibtex/grammar.d.ts	2019-11-03 21:29:52.000000000 +0100
@@ -165,8 +165,4 @@
     verbatimCommands?: string[];
 }
 
-export const AST: {
-    parse(input: string, options?: ParseOptions): AST;
-};
-
-export function parse(input: string): CSL.Data[];
+export function parse(input: string, options?: ParseOptions): AST;
